san/transform.py

```diff
@@ -0,0 +1,30 @@
+import operator
+from functools import reduce
+
+QUERY_PATH_MAP = {
+    'eth_top_transactions': ['ethTopTransactions']
+}
+
+def path_to_data(idx, query, data):
+    return reduce(operator.getitem, ['query_'+str(idx),]+QUERY_PATH_MAP[query], data)
+    """
+    With this function we jump straight onto the key from the dataframe, that we want and start from there
+    """
+
+def add_transform(function):
+    def decorator(idx, query, data):
+        if query+'_transform' in globals():
```

Comment:
Maybe one comment here is needed that if transforming function for the query exists -> the result is passed to it for another transformation.

Reply:
I've just put the comment right before the "if" statement